### PR TITLE
Nerfs holoparasites

### PR DESCRIPTION
**Pull request by Buggy123**
Instead of increasing price, I made damage dealt to holoparasites always cause their owner to take cloning damage instead of brute. This means a guard is good for protecting yourself if you get sleepy-penned and dragged into maint, but going on a rampage will quickly rack up almost unheal-able clone damage.

### CODE REVIEW

**Comment by MMMiracles** [comment]
those commits doe

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -72,18 +72,18 @@
 			loc = get_turf(summoner)
 
 
-/mob/living/simple_animal/hostile/guardian/adjustBruteLoss(amount) //The spirit is invincible, but passes on damage to the summoner
+/mob/living/simple_animal/hostile/guardian/adjustCloneLoss(amount) //The spirit is invincible, but passes on damage to the summoner
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: I think you meant to keep this as adjustBruteLoss(). We want conventional damage, not clone damage, dealt to the guardian to give clone damage to the owner.

**Comment by WJohn** [comment]
Cloning damage is extremely hard to heal, you know this is a gigantic nerf. Why not just make it so the damage dealt to the holoparasite is 1:1 with what the user takes?

**Comment by Cebasiz** [comment]
reeks of i ded pls nerf

cloning damage is unhealable without somehow conquering medbay as a one man army, which isn't possible let alone probable

damage scaling increase is fine but this is ridiculous

**Comment by Buggy123** [comment]
Alright what if the owner takes a little clone loss every hit in addition to the current brute?

**Comment by Cebasiz** [comment]
why not just increase the brute damage

who even thinks holoparasites are a problem

1. apply laser to holoparasite or user

2. ded

why even nerf them let alone into such an unusable state? they cost nearly as much as a revolver, they should be deadly.

**Comment by tedward1337** [comment]
The hell is up with this IED shit

**Comment by Durkel** [comment]
A gorrilion commits and a awful change, sign me up for some drama.

**Comment by Buggy123** [comment]
I dunno everyone has been screaming for a holo nerf and theres been like 3 PRs about it. I don't even really think holos need a nerf myself, but lots of people have been asking for it so I though I'd give it a shot.

**Comment by KorPhaeron** [comment]
>Why not just make it so the damage dealt to the holoparasite is 1:1 with what the user takes?

2 people sharing a single health pool with direct damage transfer from a (usually) melee only creature would end very quickly

**Comment by KorPhaeron** [comment]
Anyway, as always, I would rather increase the price than make them trash, and I would rather lock them out of low pop than increase the price

File: code/modules/mob/living/simple_animal/guardian/guardian.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: All damage to simple animals coverts to typeless health loss regardless, so yeah it doesn't really matter beyond the elusive "in code lore"

**Comment by Bawhoppen** [comment]
nerf for memeosite always best :+1:

**Comment by Buggy123** [comment]
Alright, now it functions normally except 20% of the damage transferred to you from your stand is taken as cloning instead of brute. This means you can heal off most of the damage taken, but continued use will slowly build up hard-to-heal cloneloss.

So if a standard parasite takes, say, 20 damage, you take that much reduced by it's damage resistance (50%), and 20% of that damage is taken as clone damage, so you take 8 brute and 2 clone loss.

**Comment by Jordie0608** [comment]
Changelog~

**Comment by KorPhaeron** [comment]
It's better now, but I still think it is fundamentally flawed to pit the security team, with infinite, regenerating ammo, against someone who is inevitably ground down by attrition. 

Traitors have poor staying power with most uplink items. Many of the items are very expensive consumables (where as the station they are antagonizing can generate more health and ammo and repairs out of thin air). 

@iamgoofball had the right idea making parapens re-usable.

**Comment by tkdrg** [comment]
This is awful and has no purpose, holoparasites are perfectly fine right now, they're not too strong on murderboner, they make it very hard to recover once you fall down, and most importantly they're fun to play with, as, and against. Of course dsay is always going to be salty when they die, but that's no excuse to nerf this.

**Comment by Bawhoppen** [comment]
I think price is still best nerf for them

**Comment by tedward1337** [comment]
Seriously why is there IED code is this PR. Stop hiding shit.

**Comment by MMMiracles** [comment]
i dont think hes hiding shit he really just doesn't know how to start a clean branch.

**Comment by Bawhoppen** [comment]
it seems like hes doing everyone on the same branch

**Comment by Buggy123** [comment]
Yeah i'm kinda bad at Github... and now it seems my reagent transfer PR also snuck in here causing a conflict... great. Before that happened there wasn't anything hidden, as you can see in the files changed area, i just did and then undid the IED PR on the same branch.

**Comment by Bawhoppen** [comment]
why didnt you make multiple branches, its simple functionality

**Comment by Buggy123** [comment]
Welp because i had to delete my fork and make a new one it's impossible for me to fix the conflict, would reopening this on a new branch violate the freeze?

**Comment by Buggy123** [comment]
I didn't understand branches until a little bit ago, and by then i had already messed with my master branch so it was too late.

**Comment by KorPhaeron** [comment]
Why would you re-open it when everyone but bawhoppen has said it is bad?

**Comment by Bawhoppen** [comment]
ye im on github ignore list

**Comment by Cheridan** [comment]
If you need to recreate that's fine I suppose. I'm not convinced any nerfs need to happen to parasites though

**Pull request closed without merging**